src/Datadog.Trace.ClrProfiler.Native/clr_helpers.h

```diff
@@ -347,6 +347,12 @@ std::vector<IntegrationMethod> FilterIntegrationsByTarget(
     const std::vector<IntegrationMethod>& integrations,
     const ComPtr<IMetaDataAssemblyImport>& assembly_import);
 
+// FilterIntegrationsByTargetAssemblyName removes any integrations which target any
+// of the specified assemblies
+std::vector<IntegrationMethod> FilterIntegrationsByTargetAssemblyName(
+    const std::vector<IntegrationMethod>& integrations,
```

Comment:
Could you rename this parameter to `integration_methods` or something similar?

Reply:
Makes sense. Renamed this one and in two other methods in 38a121c7.